packages/harden-plugin/src/types.ts

```diff
@@ -79,4 +80,16 @@ export interface HardenPluginOptions {
      * @default 'prod'
      */
     apiMode?: 'dev' | 'prod';
+    /**
+     * @description
+     * Allows you to skip the complexity check for certain requests.
```

Comment:
I think we should note that the `ctx` is _not_ the `RequestContext` object here, since I think a lot of people will make that assumption based on typical Vendure patterns.

Maybe even rename it to `context` to make it even more explicit.

Reply:
@michaelbromley  Valid concern. Fixed :-)